pkg/process/checks/net.go

```diff
@@ -263,16 +304,21 @@ func batchConnections(
 		}
 
 		cc := &model.CollectorConnections{
-			HostName:          cfg.HostName,
-			NetworkId:         networkID,
-			Connections:       batchConns,
-			GroupId:           groupID,
-			GroupSize:         groupSize,
-			ContainerForPid:   ctrIDForPID,
-			EncodedDNS:        dnsEncoder.Encode(batchDNS),
+			HostName:        cfg.HostName,
+			NetworkId:       networkID,
+			Connections:     batchConns,
+			GroupId:         groupID,
+			GroupSize:       groupSize,
+			ContainerForPid: ctrIDForPID,
+			//EncodedDNS:        dnsEncoder.Encode(batchDNS),
+			DomainDb: &model.DomainDatabase{
+				EncodedDomains: dnsEncoder.EncodeDomainDatabase(namedb),
+			},
+			EncodedDnsLookups: dnsEncoder.EncodeMapped(batchDNS),
+			IndexedDomains:    batchDomainIndexes,
 			ContainerHostType: cfg.ContainerHostType,
-			Domains:           batchDomains,
-			Routes:            batchRoutes,
+			//Domains:           batchDomains,
```

Comment:
Just putting a note as a reminder to move the commented out fields before merging.